services/auth/oauth2.go

```diff
@@ -126,7 +126,8 @@ func (o *OAuth2) userIDFromToken(tokenSHA string, store DataStore) int64 {
 // If verification is successful returns an existing user object.
 // Returns nil if verification fails.
 func (o *OAuth2) Verify(req *http.Request, w http.ResponseWriter, store DataStore, sess SessionStore) (*user_model.User, error) {
-	if !middleware.IsAPIPath(req) && !isAttachmentDownload(req) && !isAuthenticatedTokenRequest(req) {
+	if !middleware.IsAPIPath(req) && !isAttachmentDownload(req) && !isAuthenticatedTokenRequest(req) &&
```

Comment:
Could there be a comment for "why these paths can be ignored"?

Reply:
Could there be a comment for "why these paths can be ignored"?

Reply:
I thinks a comment is very good. And I found some tests failure because this change. I will fix that test first.

Reply:
3bc8a024d16b1acd164ad644bcaf86cd817b4813